assets/scss/v-tooltip.scss

```diff
@@ -0,0 +1,94 @@
+// from https://akryum.github.io/v-tooltip/#/install
+.tooltip {
+    display: block !important;
+    z-index: 10000;
```

Comment:
I almost called this out for being a malformed style, but it looks like the tooltip gets an inline position style, causing this to work. It's odd that these styles are constant as inline, and that there is `display: block !important` on a div, but it breathes